patterns/2-structured/release-process.md

```diff
@@ -0,0 +1,64 @@
+
+## Problem
+
+When a team is deciding whether to use an InnerSource project, one of their considerations is whether they trust that they can rely on the given project for an extended period. Switching the tools/projects that they are using has a cost, so they only want to make those investments when necessary and has tangible benefits.
+
+It is common practice for Open Source projects to have versioned releases, with notes documenting breaking changes, and new features along with either a published binary or link to a Docker image. This practice may not be as transparent or well documented/visible for InnerSource projects, modules, etc.
+
+InnerSource projects that don't have a published artifact or release process reduces trust. Teams won't know when they can expect the next release, when breaking changes are introduced, etc.
+
+## Context
+
+Large organizations produce a lot of internal software, much of which could be reused by teams across the company. Operational tooling, software libraries, and Infrastructure as a Service modules are all examples of this. Most large organizations don't publish internal software to be consumed by other teams in the company, however. This can occur either because they are too busy provide documentation or don't realize it's value to others.
+
+An internal or public source repository should be available where source code is stored, but teams lack a process for making software consumable by outside teams.
+
+As an organization grows and more internal software is written, the value of this pattern grows.
+
+## Forces
+
+### Difficult for organizations that don't have a central CI/CD system
+
+For organizations that don't provide engineers a centralized CI/CD system, automating a build and release process can be challenging. The team may need to stand up their own tool (Jenkins, Drone, etc). Without a CI/CD system, builds and release notes can still be produced, however, it may require a local build of the software and manual upload to whichever tool is hosting build artifacts.
+
+### Added burden of publishing release notes
+
+In addition to building your source code, writing release notes can be tedious without the ability to auto-generate a list of git commits. This would be left for someone to do manually, in addition to writing more high level details on a release.
+
+### Increased difficulty without a location to host artifacts
+
+If a company does not provide a centralized location for storing build artifacts (jars, npm modules, etc.) and docker images, engineers may be left deciding for themselves where is appropriate to store versioned software. Tools like GitHub provide this for you, however, if a company is not using one of these popular tools, this could pose a burden.
+
+## Solution
+
+- CI/Delivery Pipeline is located within your repo to build artifacts (binary, docker image, jar, etc)
```

Comment:
```suggestion
By providing clear **release notes** and a published artifact you increase people's confidence that you are publishing a quality product that they can rely on.

The following are key elements to achieve this:

- CI/Delivery Pipeline is located within your repo to build artifacts (binary, docker image, jar, etc)
```

Many people might jump straight from reading the **Problem** section to reading the **Solution** section. In terms of formatting I would therefore start the Solution with a bit of prose, before using bullets.

Left a draft of how that might look like above but please adapt to your liking.